int32 status
string robot_state
string robot_case
int32[] pos
int32[] enemy1_pos
int32[] enemy2_pos
int32[] ally_pos
bool is_blocked
int32 servo_state
int32 stepper_state
int32 hand_state
int32 action_done
int32 kill_mission
int32 goal_covered_counter


